rtrlib/transport/ssh/ssh_transport.h

```diff
@@ -45,6 +45,7 @@
  *	  is made. The returned socket is expected to be ready for use (e.g.
  *	  in state established), and must use a reliably stream-oriented transport.
  *	  When new_socket() is used, host, port, and bindaddr are not used.
+ * @param connect_timeout Time in seconds to wait for a succeccfull connection. Defaults to 30 seconds
  */
```

Comment:
nit: s/succeccfull/successfull

Reply:
actually it is [sucessful](https://www.merriam-webster.com/dictionary/successful), but it is wrong either way, so thank for pointing that out.